dbms/src/test/java/org/polypheny/db/misc/DataMigratorTest.java

```diff
@@ -146,7 +166,7 @@ public void alternativeCommandsTest() throws SQLException {
                 try {
                     statement.executeUpdate( "INSERT INTO datamigratortest VALUES (1,5,'foo')" );
 
-                    // Add data store
+                    // Add data storeId
```

Comment:
Looks like a find and replace error

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520476075